force_gromacs/pipeline/tests/test_gromacs_pipeline.py

```diff
@@ -96,43 +133,7 @@ def test_append(self):
             keys
         )
 
-    def test__validate_steps(self):
-
-        dummy_command = DummyCommand()
-
-        with self.assertRaises(TraitError):
-            self.pipeline.append(
-                ('not_a_command', dummy_command)
-            )
-        self.assertEqual(4, len(self.pipeline))
-
-        dummy_command.run = True
-        dummy_command.bash_script = True
-        dummy_command.dry_run = True
-
-        with self.assertRaises(TraitError):
-            self.pipeline.append(
-                ('not_a_command', dummy_command)
-            )
-        self.assertEqual(4, len(self.pipeline))
-
-        dummy_command = DummyCommand_2()
-
-        with self.assertRaises(TraitError):
-            self.pipeline.append(
-                ('not_a_command', dummy_command)
-            )
-        self.assertEqual(4, len(self.pipeline))
-
-        dummy_command = DummyCommand_3()
-
-        with self.assertRaises(TraitError):
-            self.pipeline.append(
-                ('not_a_command', dummy_command)
-            )
-        self.assertEqual(4, len(self.pipeline))
```

Comment:
This code was left over from when `GromacsPipeline` was not using `traits` for type checking